themes/origins/snippets/product-card.liquid

```diff
@@ -0,0 +1,50 @@
+{% comment %}
+  Renders a product card
+
+  Accepts:
+  - product: {Object} Product Liquid object (optional)
+  - lazy_load: {Boolean} Image should be lazy loaded. Default: true (optional)
+  - placeholder_image: {String} The placeholder image to use when no product exists. Default: 'product-apparel-2' (optional)
+
+  Usage:
+  {% render 'card-product', lazy_load: true %}
+{% endcomment %}
+
+{{ 'product-card.css' | asset_url | stylesheet_tag }}
+
+{% style %}
+  .product-card {
+    --border-radius: {{ settings.product_card_border_radius }}px;
+    --badge-text-color: {{ settings.badge_text_color.hex }};
+    --badge-background-color: {{ settings.badge_background_color.hex }};
+  }
+{% endstyle %}
+
+<div class="product-card {{ settings.product_card_style }}">
```

Comment:
looking at the previous components, customization options were added as `data-attr`

```suggestion
<div class="product-card" data-style="{{ settings.product_card_style }}">
```